.github/workflows/nightly.yaml

```
@@ -0,0 +1,39 @@
+name: Nightly
+on:
+  schedule:
+    # Times in UTC (PST+8), this translates to running Mon-Fri at midnight/1am (depending on DST)
+    - cron: 0 8 * * 1-5
+
+jobs:
+  plugins_test:
+    name: Nightly Plugin Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        linter-version: [Snapshots, Latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+        # TODO(Tyler): Determine if we want to wipe the download cache and start fresh each night.
+
+      - name: Cache tool downloads
+        uses: actions/cache@v3
+        with:
+          path: /tmp/plugins_testing_download_cache
+          # No need to key on trunk version unless we change how we store downloads.
```

```suggestion
          # No need to key on trunk version or trunk.yaml unless we change how we store downloads.
```

in comparison to https://github.com/trunk-io/trunk-action/blob/2cda8a0c9923423307c2c3efe676a4c4614e46e6/action.yaml#L88-L91